x-pack/plugins/alerting/server/routes/suggestions/fields_rules.ts

```diff
@@ -53,10 +53,11 @@ export function registerFieldsRoute(
           });
         }
         const indices = [ALERTING_CASES_SAVED_OBJECT_INDEX];
-        const { elasticsearch } = await context.core;
+        const { elasticsearch, uiSettings } = await context.core;
 
         const indexPatternsFetcherAsInternalUser = new IndexPatternsFetcher(
-          elasticsearch.client.asInternalUser
+          elasticsearch.client.asInternalUser,
```

Comment:
This means the field list will be limited by the configured tiers. Seems desirable to me, a nice performance boost.